libs/sdk-ui-charts/api/sdk-ui-charts.api.md

```diff
@@ -189,6 +189,7 @@ export interface IChartConfig {
     colorMapping?: IColorMapping[];
     colorPalette?: IColorPalette;
     colors?: string[];
+    continuousLine?: IContinuousLineConfig;
```

Comment:
Please prepare storybook screenshot test for this configuration. For line chart with this option enabled, and consider also having one screenshot for combo chart with this config (optional).

Reply:
I have added the storybook test. Thanks!!